tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -0,0 +1,28 @@
+# std
+import unittest
+from pathlib import Path
+
+# project
+from src.chia_log.handlers.wallet_added_coin_handler import WalletAddedCoinHandler
+from src.notifier import EventType, EventService, EventPriority
+
+
+class TestWalledAddedCoinHandler(unittest.TestCase):
+    def setUp(self) -> None:
+        self.handler = WalletAddedCoinHandler()
+        self.example_logs_path = Path(__file__).resolve().parents[1] / "logs/wallet_added_coin"
+
+    def testNominal(self):
+        with open(self.example_logs_path / "nominal.txt") as f:
```

Comment:
Encoding needs to specifically set to UTF-8 or unit tests will fail under Windows